insitu/views/requirement_product_requirement.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.core.urlresolvers import reverse
+from django.views.generic import CreateView, UpdateView, DeleteView
+
+from insitu import models
+from insitu import forms
+
+
+class RequirementProductRequirementAdd(CreateView):
```

Comment:
I'll put this next to the ProductRequirement views. It's the same model, even though presentation logic is different.